app/models/comment.rb

```diff
@@ -331,4 +331,82 @@ def self.gmail_parsed_mail(mail_doc)
   def trimmed_content?
     comment.include?(COMMENT_FILTER)
   end
+
+  def self.receive_tweet
+    comments = Comment.where.not(tweet_id: nil)
+    if comments.any?
+      receive_tweet_using_since comments
+    else
+      receive_tweet_without_using_since
+    end
+  end
+
+  def self.receive_tweet_using_since(comments)
+    comment = comments.last
+    since_id = comment.tweet_id
+    tweets = Client.search("to:PublicLab", since_id: since_id).collect do |tweet|
+      tweet
+    end
+    tweets = tweets.reverse
+    check_and_add_tweets tweets
+  end
+
+  def self.receive_tweet_without_using_since
+    tweets = Client.search("to:PublicLab").collect do |tweet|
```

Comment:
Ah, so hard-coded to PublicLab account. Is there a way we could specify this in a config file or environment variable? I think an env variable may be appropriate so that this could be changed and isn't hard-coded, no?

Reply:
Yeah! I will change it.